package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "console-gitops-plugin",
+  "description": "OpenShift Console plugin for GitOps",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "build-ORIG": "yarn clean && NODE_ENV=production yarn ts-node ./node_modules/.bin/webpack --mode=production",
+    "build-dev-ORIG": "yarn clean && yarn ts-node ./node_modules/.bin/webpack --mode=development",
+    "http-server-ORIG": "./http-server.sh ./dist",
```

Comment:
What's the reason for changing these to -ORIG ?

Reply:
Will delete those.